app.py

```diff
@@ -17,6 +17,12 @@
 from flask import Flask, render_template, request, jsonify
 from chatbot import get_opening_message, get_choice, get_topic, match, narrow, ask, end
 import json
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--port", default=8000)
```

Comment:
The ports needs to be `8088` or to match the port numbers in max-question-answering-web-app.yaml, no?